iOS-sample-app/AppDelegate.swift

```diff
@@ -3,6 +3,7 @@ import UIKit
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
+    var window: UIWindow?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Note that you do not need to manually start the Contentsquare SDK yourself, it
```

Comment:
Could you use a stronger language here, saying that they *should not* start the SDK by hand ?

Reply:
dons